libs/luci-lib-jsonc/src/jsonc.luadoc

```diff
@@ -121,10 +121,22 @@ parser:set({ "some", "data" })`
 ]]
 
 ---[[
-Serialize current parser state as JSON.
+Generate an ltn12-compatible sink.
 
 @class function
 @sort 4
+@name parser.sink
+@return Returns a function that can be used as an ltn12 sink.
+@usage `parser = luci.jsonc.new()
+ltn12.pump.all(ltn12.source.file(io.input()), parser:sink())
+print(parser:get())`
+]]
```

Comment:
@karlp See http://htmlpreview.github.io/?https://github.com/openwrt/luci/blob/master/documentation/api/modules/luci.jsonc.html and https://htmlpreview.github.io/?https://raw.githubusercontent.com/openwrt/luci/master/documentation/api/modules/luci.jsonc.parser.html